documentation/API-design-guidelines.md

```diff
@@ -777,25 +777,37 @@ In the following, we elaborate on the existing client errors. In particular, we
 |       503        |      `UNAVAILABLE`       | Service Unavailable.                                                                                          | Service is not available. Temporary situation usually related to maintenance process in the server side                                  |
 |       504        |        `TIMEOUT`         | Request timeout exceeded.                                                                                     | API Server Timeout                                                                                                                       |
 
-> _NOTE 1: When no login has been performed or no authentication has been assigned, a non-descriptive generic error will always be returned in all cases, a `UNAUTHENTICATED` 401 “Request not authenticated due to missing, invalid, or expired credentials.” is returned, whatever the reason._
+> _NOTE 1: When no login has been performed or no authentication has been assigned, a non-descriptive generic error will always be returned in all cases, an `UNAUTHENTICATED` 401 “Request not authenticated due to missing, invalid, or expired credentials.” is returned, whatever the reason._
 
 > _NOTE 2: A {{SPECIFIC_CODE}}, unless it may have traversal scope (i.e. re-usable among different APIs), SHALL follow this scheme for a specific API: {{API_NAME}}.{{SPECIFIC_CODE}}_
 
-### 6.2 Error Responses - Device Object
 
-This section is focused in the guidelines about error responses around the concept of `device` object.
+**Mandatory Errors** to be **documented in CAMARA API Spec YAML** are the following:
+
+- For event notifications flow, the ones defined in [notification-as-cloud-event.yaml](/artifacts/notification-as-cloud-event.yaml)
+- For event subscriptions APIs, the ones defined in [event-subscription-template.yaml](/artifacts/camara-cloudevents/event-subscription-template.yaml)
```

Comment:
I would prefer to include all normative requirements in the guidelines and use artifacts as illustrative files or templates.
It looks that we have respective requirements in chapter 12:
```suggestion
**Mandatory Errors** to be **documented in CAMARA API Spec YAML** are the following:

- For event subscriptions APIs, the ones defined in [12.1 Subscription](#error-definition-for-resource-based-explicit-subscription)
- For event notifications flow, the ones defined in [12.2 Event notification](#error-definition-for-event-notification)
```

Currently these points in Design Guidelines are not changed in PR #313, 
I propose to introduce these changes within this PR #329 after merging PR #313 

WDYT @bigludo7 @PedroDiez  ?

Reply:
It is fine, to be adressed after PR#313